README.md

```diff
@@ -1 +1,54 @@
-# ngx-keyboard-shortcuts
\ No newline at end of file
+## NGX Keyboard Shortcuts
+
+This project is to help you create keyboard shortcuts in your angular applications. When we say keyboard shortcuts we are referring to something like ctrl+c which would allow you to copy whatever you highlighted. 
+In this case you will be able to use our application to create keys that can do whatever you desire. 
+
+## Getting Started 
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+If you are intending to use this repository you should be familiar with ``angular``, and will need to install angular on your machine. The command needed to install angular is:
+``npm install -g @angular/cli`` 
+
+### Installing Our Package 
+To install our package you would use the node package manager.
+The command would be  
+``npm install ngx-keyboard-shortcuts``
+Then using the command 
+``npm start`` 
+You will be able to deploy this service in **localhost:4200** to see what you can do with the application. 
+
+
+### Demos
+Once you open **localhost:4200** in your search bar you will see we have put several live demos for you to try on your own in our Demo Application. 
+These will show you how you can use what we created in your own application and hopefully give you some ideas of new was to implement this. 
+
+## Running Coverage Tests
+In order to run tests using **Karma** and **Jasmine** you will have to type the command 
+``npm run test``
+This will then output the percentage of **Statements**, **Branches**, **Functions**, and **Lines** covered in your testing. 
+What these tests actually test is if the code gets executed will it do what it is supposed to do. The most important thing when testing is making sure that the **Statements** are 100% covered. 
+
+## Built With 
+* [Angular]([https://angular.io/](https://angular.io/)) - The web framework used 
+* [Npm]([https://www.npmjs.com/get-npm](https://www.npmjs.com/get-npm)) - The package manager
+
+## Version
+Not released 
+
+## Contributing 
+
+Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Authors 
```

Comment:
make this section entitled "contributors" and have a link to the packages contributors page and a short text something like "Thank you to the contributors on this project."

Reply:
I deleted the contributing and took out the authors. Making a single contributors title.